components/Layout/Navigation.js

```diff
@@ -26,6 +26,8 @@ class Navigation extends React.Component {
       <nav className="mdl-navigation" ref="root">
         <Link className="mdl-navigation__link" to="/">Home</Link>
         <Link className="mdl-navigation__link" to="/about">About</Link>
+        <Link className="mdl-navigation__link" to="/blog/2016/07/01/Post-1">Post 1</Link>
+        <Link className="mdl-navigation__link" to="/blog/2016/07/01/Post-2">Post 2</Link>
```

Comment:
I was following [Jekyll's article URL conventions](https://jekyllrb.com/docs/permalinks/) - I'm pretty sure it's SEO friendly since it's descriptive of the article